README.md

```diff
@@ -52,6 +52,7 @@ stratified those patients with long and serious hospital stays under the long-st
 | [Configuration Files](config/README.md) | Overview of provided configuration files |
 | [Production Build Configuration Files](deploy/README.md) | Overview of the configuration files provided for production build Docker containers |
 | [Training](training/README.md) | Description of the training process for the models used in the LTSS API |
+| [Generating fake data](fake_data_generation/README.md) | Description of how to generate fake data to test the setup and running of the repo |
```

Comment:
I would move this higher in the table as logically this will come before testing